polygeist/lib/Dialect/Polygeist/Analysis/MemoryAccessAnalysis.cpp

```diff
@@ -713,21 +896,6 @@ MemoryAccess<OpTy>::classifyMemoryAccess(DataFlowSolver &solver) const {
 // MemoryAccessAnalysis
 //===----------------------------------------------------------------------===//
 
-/// Walks up the parents and records the ones with the specified type.
```

Comment:
Moved up with the other utility functions (see lines 93-106). I've also addressed a comment @whitneywhtsang made on this code in the previous PR in this series.